scripts/install/install_macos.sh

```diff
@@ -272,6 +278,36 @@ check_prereqs()
   decrease_indent
 }
 
+# Test connection to BindPlane if it was specified
+connection_check()
+{
+  info "Help Me Obi-Wan Kenobi!"
+  if [ $check_bp_url == "true" ] ; then
+    if [ -n "$opamp_endpoint" ]; then
+      HTTP_ENDPOINT="$(printf "${opamp_endpoint}" | sed 's#ws#http#' | sed 's#/v1/opamp$##')"
+      info "Testing connection to BindPlane: $fg_magenta$HTTP_ENDPOINT$reset..."
+
+      if curl -s ${HTTP_ENDPOINT} > /dev/null; then
+        succeeded
+      else
+        failed
+        warn "Connection to BindPlane has failed. Do you wish to continue installation?"
+        increase_indent
+        printf "${fg_yellow}Do you wish to continue installation?${reset}  "
+        prompt "n"
+        decrease_indent
+        read -n 1 -p "" input
```

Comment:
The `-n` and `-p` flags aren't posix compliant, do we need these flags?

Reply:
They are not. This was an opinionated take, and makes it so you can just hit y or n without pressing enter. If we remove these flags, you'll need to hit enter after y or n.

Reply:
If you still want that old behavior, I found this on stack overflow:
```sh
# Adapted from https://unix.stackexchange.com/questions/464930/can-i-read-a-single-character-from-stdin-in-posix-shell
read_byte() 
{
  if [ -t 0 ]; then
    # disable icanon mode if this is a terminal
    saved_tty_settings=$(stty -g)
    stty -icanon min 1 time 0
  fi
 
  by=$(dd bs=1 count=1 2> /dev/null)
  
  printf "%s" "$by"
 
  if [ -t 0 ]; then
    # revert tty settings to their original values
    stty "$saved_tty_settings"
  fi
}
```

Works like:
```sh
input=$(read_byte)
```

Reply:
Thanks, but I'll just do without it. I don't like when scripts modify tty settings. If you ctrl-c out of them at just the wrong time, your tty settings are borked. ;)